common/mariadb/scripts/mariadb-set-root-password.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -eo pipefail
+shopt -s nullglob
+set -o xtrace
```

Comment:
Shouldn't that be disabled? Otherwise the password will appear in the logs.

Reply:
Yes, it must be disabled, missed the `set +x` below.